src/docs/product/alerts/create-alerts/metric-alert-config.mdx

```diff
@@ -13,11 +13,14 @@ Sentry provides several configuration options to create a metric alert based on
 To create a metric alert you first need to choose a metric type. For some alert types the function is built into the alert and for others you can choose functions and parameters to apply to it. For example, if you select “Users Experiencing Errors”, that translates to the function, `count_unique(user.id)`. Since editing this function would change the nature of the alert, it is not editable and thus hidden. On the other hand, if you select “Largest Contentful Paint” the measurement used is `measurement.lcp`, you also need to choose a function, e.g. `p75()`, for a combined metric function of `p75(measurement.lcp)`.
 
 ### Metrics Types for Alerting
-
+#### Errors
 - Number of Errors
 - Users Experiencing Errors
+#### Sessions
 - Crash Free Session Rate
 - Crash Free User Rate
+#### Performance
+For performance alerts, [metrics-based](/product/sentry-basics/search/searchable-properties/#metrics-properties) data is the default. If you make a selection in configuring your alert that's not metrics-compatible, then your alert will use event-based data and a message saying this will be displayed.
```

Comment:
```suggestion
For performance alerts, [metric-backed](/product/sentry-basics/search/searchable-properties/#metrics-properties) data is the default. If you make a selection in configuring your alert that's not metrics-compatible, then your alert will use indexed events, or event-based data, and a message saying this will be displayed.
```